lib/src/core/utils/string_constants.dart

```diff
@@ -1,3 +1,23 @@
 abstract class StringConstants {
   static const String homeTitle = 'Home';
```

Comment:
Review this file, try to keep here only the constants that are used in more than one place, and leave the specific const inside each file that uses it

Reply:
Done